an-apluss/epic-mail
# Ft passwordreset 164297425

#### What does this PR do?
- This display a page to take user's input, specifically email address

#### Description of Task to be completed?
- Create a user interface to accept user email address

#### How should this be manually tested?
- Ensure you have access to internet
- Ensure you have web browser installed
- Click [link]( https://an-apluss.github.io/epic-mail/UI/passwordreset.html )

#### What are the relevant pivotal tracker stories?
- PT Story link - ( `[164297425](https://www.pivotaltracker.com/story/show/164297425)`)

#### Screenshots 
![password reset - epic -](https://user-images.githubusercontent.com/45961117/53880016-2f603780-4010-11e9-8333-f9a269d80583.png)
